### DISCUSSION TITLE
新一代iPad Pro

### DISCUSSION THREAD

**Discussion by brian431** [1101-GS3073-C/Python-Community-GS3073-C#481] [科技新知]
和大家的猜測一樣搭載了M2晶片，甚至推出了全新的Apple Pencil懸浮感應功能。

https://technews.tw/2022/10/18/m2-ipad-pro-5/

**Comment by lx3211** [comment]
Apple Pencil 2 的懸浮功能真是令人好奇，可以在12公釐的時候就被偵測到，不知道會不會很容易誤觸？

**Comment by Chenszuhong** [comment]
感覺搭載M2不知道是有好有壞！？，雖然效能變好但可能影響使用的時間

**Comment by xCk27x** [comment]
【Apple Pencil 最多可在距顯示器上方 12 公釐時被偵測到，方便使用者在標示記號前先行預覽。這也讓使用者能夠更精準地進行素描或繪畫，並更輕鬆地使用 Apple Pencil 做任何事。】
我覺得這功能很實用耶! 對於下筆精準度會有很大的提升
常常需要精密操作時卻一直點錯位置😢 真的會很崩潰==

**Comment by williamyen0623** [comment]
自己是用搭載M1的ipad pro, 最近覺得蠻耗電的, 不知道搭載M2晶片能不能改善這方面的問題, 還是只是提升性能而已
apple pencil 懸浮功能真的好酷, 蘋果把拔從來不讓我們失望

**Comment by Ben6uo** [comment]
太強了 可惜我沒有這一個錢錢阿

**Comment by HsuChen88** [comment]
原地拋售iPad Air 5，原本的電池續航力就是一缺點了，若再加上M2的耗能不知續航力如何?

**Comment by 0212Jerry** [comment]
自己覺得跟M1的ipad pro 比起來，功能上基本沒什麼差別，而且M1跟M2的差距在使用上甚至有些人感覺不出來，所以如果是要升級iPad Pro 的我建議可以再等。但如果是更久的機型的話就可以考慮

**Comment by Valentinaa0214** [comment]
Apple Pencil 2 的懸浮功能感覺會是利用iPad創作的設計師們的一大福音，Apple Pencil 的功能真是越來越強大了，未來可能沒有副廠的觸控筆能夠取代原廠的功能了

**Comment by Vincent20030314** [comment]
感覺是一個讓人期待的功能，但可惜我已經有ipad跟apple pencil了。
